app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/Router.scala

```diff
@@ -19,12 +19,21 @@ package uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers
 import play.api.mvc.Call
 import uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.{JourneyBindable, routes => claimRoutes}
 import uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers
-import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.MrnJourney
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.{BasisOfClaim, MrnJourney}
 import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.MrnJourney.MrnImporter
 
 trait JourneyTypeRoutes extends Product with Serializable {
   val subKey: Option[String]
   val journeyBindable: JourneyBindable
+
+  def nextPageForBasisForClaim(basisOfClaim: BasisOfClaim): Call =
```

Comment:
I know these methods are very simple, but i think there is strong business value in adding descriptive unit tests for all of them. 

eg

"The next page after basis of claim" should be 
- "enter duplicate reference number when basis of claim has duplicate entry selected"
- "enter commodities details when basis of claim doesn't have duplicate entryselected"

Reply:
Done